QGL/ChannelLibraries.py

```diff
@@ -389,12 +389,16 @@ def new_source(self, label, model, address, power=-30.0, frequency=5.0e9, refere
         self.add_and_update_dict(thing)
         return thing
 
-    def set_control(self, qubit_or_edge, transmitter, generator=None):
+    def set_control(self, qubit_or_edge, transmitter, generator=None, aps1_chan=None):
         quads   = [c for c in transmitter.channels if isinstance(c, Channels.PhysicalQuadratureChannel)]
         markers = [c for c in transmitter.channels if isinstance(c, Channels.PhysicalMarkerChannel)]
 
         if isinstance(transmitter, Channels.Transmitter) and len(quads) > 1:
-            raise ValueError("In set_control the Transmitter must have a single quadrature channel or a specific channel must be passed instead")
+            #raise ValueError("In set_control the Transmitter must have a single quadrature channel or a specific channel must be passed instead")
+            print("Warning: In set_control the Transmitter must have a single quadrature channel if using APS2. Assuming APS1.")
```

Comment:
I'd use `warn` from `warnings` module

Reply:
After talking to Graham, we kept the error message. It turns out you can access the main transmitter back from the single channel, so we do not need to pass the whole APS1 transmitter.